doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -112,19 +112,30 @@ At this time, we do not make any guarantees about the exactness or freshness of
-the frontiers the coordinator is aware of.
+The `mz_cluster_replica_frontiers` table describes the per-replica frontiers of
+each source, sink, materialized view, index, and subscription in the system,
+as observed from the coordinator.
+
+[`mz_frontiers`](#mz_frontiers) is similar to this table, but `mz_frontiers`
+reports the frontier for each object aggregated across all replicas, rather than
+per replica. Note also that `mz_frontiers` contains frontiers for objects
+that are not associated with a replica, like tables.
+
+[`mz_compute_frontiers`](#mz_compute_frontiers) is similar to this table, but
+`mz_compute_frontiers` reports the frontiers known to the active compute
+replica, while this table reports the frontiers the coordinator is aware of.
+Note also that `mz_compute_frontiers` is restricted to compute objects (indexes,
+materialized views, and subscriptions) while this table contains storage objects
+(sources and sinks) as well.
 
 At this time, we do not make any guarantees about the freshness of these numbers.
 
 <!-- RELATION_SPEC mz_internal.mz_cluster_replica_frontiers -->
-| Field         | Type             | Meaning                                                                                                                                                  |
-| ------------- | ------------     | --------                                                                                                                                                 |
-| `replica_id`  | [`text`]         | The ID of a cluster replica.                                                                                                                             |
-| `export_id `  | [`text`]         | The ID of the index, materialized view, or subscription that created the dataflow. Corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports). |
-| `time`        | [`mz_timestamp`] | The next timestamp at which the output may change.                                                                                                       |
+| Field         | Type             | Meaning                                                                                                                                                                                       |
+| ------------- | ------------     | --------                                                                                                                                                                                      |
+| `replica_id`  | [`text`]         | The ID of a cluster replica.                                                                                                                                                                  |
+| `object_id `  | [`text`]         | The ID of the source, sink, index, materialized view, or subscription. If an index, materialized view, or subscription, corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports). |
```

Comment:
Should we just drop the "corresponds to" part, like we do for `mz_frontiers`? It seems like this is now more confusing than it is helpful. For example, people might wonder if for sources and sinks the `object_id` corresponds to nothing. Or why we point out `mz_compute_exports` here, rather than `mz_indexes`/`mz_materialized_views`/`mz_subscriptions`.